contract/vm_dummy/test_files/feature_isolation.lua

```diff
@@ -22,8 +22,10 @@ function override_functions()
     -- override contract.balance
     contract.balance = function() return "123" end
 
-    -- override the __add metamethod on bignum module
-    getmetatable(bignum.number(0)).__add = function(x,y) return x-y end
+    if getmetatable ~= nil then
+      -- override the __add metamethod on bignum module
+      getmetatable(bignum.number(0)).__add = function(x,y) return x-y end
```

Comment:
I don't review deeper code about getmemtable, but it doesn't look like addtion rather substraction. Anyway, if this code did not cause any problem, we can leave this as is, since this commit don't change the code.

Reply:
yes, this test overrides the add (`+`) operator with a subtraction (`-`) to check if that feature of overriding functions is possible. the feature is disabled by this PR